settings.gradle.kts

```diff
@@ -22,7 +22,7 @@ listOf("fiat", "kork").forEach { prj ->
   }
 }
 
-enableFeaturePreview("VERSION_ORDERING_V2")
+// enableFeaturePreview("VERSION_ORDERING_V2")
```

Comment:
Can you add a comment here that mentions what we're waiting on to re-enable this?